packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -324,6 +324,14 @@ export abstract class FluidDataStoreContext extends TypedEventEmitter<IFluidData
         this._tombstoned = true;
     }
 
+    public revive() {
```

Comment:
Consider renaming this to `untombstone` so it reflects that it is related to and is opposite of `tombstone`. 

Alternatively, you can have a funtion `setTombstoneState(tombstone: boolean)` which does both setting and unsetting the value. This one may be slightly better as it consolidates the logic and we have other similar methods such as `setConnectionState`.

Reply:
Created `setTombstone()` guess I forgot to say state, but I think it should be fine